test/internal/List.js

```diff
@@ -0,0 +1,105 @@
+'use strict';
```

Comment:
This file was largely copied from [sanctuary-type-classes][1].


[1]: https://github.com/sanctuary-js/sanctuary-type-classes/blob/v6.0.0/test/List.js